Automattic/sensei
# Update draft, unsaved lesson post status display

Depends on #3650 as that one has some spacing changes to the lesson block.

### Changes proposed in this Pull Request

* Make the post status for the lessons always visible if they are unsaved or draft
* Remove it's background, move below lesson title — shrinking the editable input area for the lesson title made it a bit harder to click the input

### Testing instructions

* Create a course with outline, add new lessons
* Check that their unsaved or, after saving, draft statuses are displayed



### Screenshot / Video
<img width="629" alt="image" src="https://user-images.githubusercontent.com/176949/95114642-8bd38d80-0744-11eb-846a-a890a625fcf1.png">
